internal/api/v1/serviceplans.go

```diff
@@ -0,0 +1,64 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/epinio/epinio/helpers/kubernetes"
+	"github.com/epinio/epinio/internal/cli/clients"
+	"github.com/epinio/epinio/internal/services"
+	"github.com/julienschmidt/httprouter"
+)
+
+type ServicePlansController struct {
+}
+
+func (spc ServicePlansController) Index(w http.ResponseWriter, r *http.Request) {
+	params := httprouter.ParamsFromContext(r.Context())
+	org := params.ByName("org")
+	serviceClassName := params.ByName("serviceclass")
+
+	cluster, err := kubernetes.GetCluster()
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	gitea, err := clients.GetGiteaClient()
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+
+	exists, err := gitea.OrgExists(org)
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	if !exists {
+		http.Error(w, fmt.Sprintf("Organization '%s' does not exist", org),
+			http.StatusNotFound)
+		return
+	}
+
+	serviceClass, err := services.ClassLookup(cluster, serviceClassName)
```

Comment:
and again the org does not play any role at all in listing the plans of a class.

Reply:
Done.